style.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,300;0,400;1,300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+
+  /* overflow: hidden; */
+}
+
+.toolbar_container {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  max-width: 100%;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  gap: 180px;
+  height: 17vh;
+}
+
+.Title_logo {
+  color: #fff;
+  font-weight: bold;
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+}
+
+.main_container {
+  width: 100%;
+  height: 100%;
+  background-color: #1c1a19;
+  background-image: url('./images/Illustration.png');
+  background-repeat: no-repeat;
+  background-position: 200px 170px;
+}
+
+.main_container,
+.text_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  align-content: center;
+  color: #fff;
+  text-align: justify;
+  gap: 10px;
+}
```

Comment:
- [ ] In the figma design, the background-image goes all the way down. you can achieve that by just adding `padding-bottom: 5rem;` here.
look at the difference between your deisgn and figma design
<img width="491" alt="Screenshot 2023-07-05 at 10 07 49 PM" src="https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/96683843/5d5fee94-e6ee-4f29-a630-83e3e1434c76">

<img width="281" alt="Screenshot 2023-07-05 at 10 07 03 PM" src="https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/96683843/c0e9273f-7a0c-47bc-800e-56bb8651d346">

Reply:
Fixed.

Reply:
this is also fixed